conda/environments/all_cuda-118_arch-x86_64.yaml

```diff
@@ -24,8 +24,8 @@ dependencies:
 - ipython
 - ipywidgets
 - libcudf==24.8.*,>=0.0.0a0
-- libcuspatial-tests==24.8.*
-- libcuspatial==24.8.*
+- libcuspatial-tests==24.8.*,>=0.0.0a0
+- libcuspatial==24.8.*,>=0.0.0a0
```

Comment:
These were pulled in by the updates from https://github.com/rapidsai/rapids-metadata/pull/24.

This is a good change that should get onto 24.08 even if this PR ends up getting re-targeted to 24.10.